source/s3_client.c

```diff
@@ -752,14 +753,6 @@ static struct aws_s3_meta_request *s_s3_client_meta_request_factory_default(
 
     /* Call the appropriate meta-request new function. */
     if (options->type == AWS_S3_META_REQUEST_TYPE_GET_OBJECT) {
-
-        /* If the initial request already has partNumber, the request is not splittable(?). Treat it as a Default
-         * request.
-         * TODO: Still need tests to verify that the request of a part is splittable or not */
-        if (aws_http_headers_has(initial_message_headers, aws_byte_cursor_from_c_str("partNumber"))) {
```

Comment:
why delete this part? And have you tried to use this branch for aws-crt-cpp and see the CI is happy or not?

Reply:
mistake, fixing now. How do I run CI on crt-cpp with s3 set to this branch?

Reply:
BYO_CRYPTO works on crt-cpp with s3 on this branch.
https://github.com/awslabs/aws-crt-cpp/runs/3107882821?check_suite_focus=true